legacy/routes/lros.js

```diff
@@ -84,6 +86,95 @@ var lros = function (coverage) {
       .end('{ "properties": { "provisioningState": "Succeeded"}, "id": "100", "name": "foo" }');
   });
 
+  router.get("/patch/201/retry/onlyAsyncHeader", function (req, res, next) {
+    res
+      .status(200)
+      .type("json")
+      .end(
+        '{ "properties": { "provisioningState": "Succeeded"}, "id": "/lro/patch/201/retry/onlyAsyncHeader", "name": "foo" }',
+      );
+  });
+
+  router.patch("/patch/201/retry/onlyAsyncHeader", function (req, res, next) {
+    var headers = {
+      "Azure-AsyncOperation": getRequestBaseUrl(req) + "/lro/patch/201/retry/onlyAsyncHeader/operationStatuses/201",
+    };
+    res
+      .set(headers)
+      .status(201)
+      .type("json")
+      .end(
+        '{ "properties": { "provisioningState": "Accepted"}, "id": "/lro/patch/201/retry/onlyAsyncHeader", "name": "foo" }',
+      );
+  });
+
+  router.get("/patch/201/retry/onlyAsyncHeader/operationStatuses/201", function (req, res, next) {
+    coverage["LROPatch201WithAsyncHeader"]++;
+    var status = "Accepted";
+    if (coverage["LROPatch201WithAsyncHeader"] === 2) {
+      status = "Updating";
+    } else if (coverage["LROPatch201WithAsyncHeader"] === 3) {
+      status = "Succeeded";
+    }
+    res
+      .status(200)
```

Comment:
Should not this be 202 since the operation is still pending?
```suggestion
      .status(202)
```

Reply:
Hi @deyaaeldeen , thanks for your review. I use 200 as status code because of this [doc](https://github.com/Azure/azure-resource-manager-rpc/blob/4e27f78ea366c11b59e8acced9d3bddf6355955a/v1.0/async-api-reference.md?plain=1#L176).